python_modules/dagster/dagster/_core/definitions/schedule_definition.py

```diff
@@ -272,12 +276,15 @@ def merge_resources(self, resources_dict: Mapping[str, Any]) -> "ScheduleEvaluat
         check.invariant(
             self._resources is None, "Cannot merge resources in context that has been initialized."
         )
+        from dagster._core.execution.build_resources import wrap_resources_for_execution
+
+        wrapped_resources_defs = wrap_resources_for_execution(resources_dict)
```

Comment:
actually having the `wrapped_resources_defs` makes the line below more confusing. I would just inline